### PR TITLE
DBZ-4298 Support non-JDBC logical values in keys for incremental snap…

### PR DESCRIPTION
**Pull request by jpechane**
…shots
https://issues.redhat.com/browse/DBZ-4298

### CODE REVIEW

File: debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/IncrementalSnapshotIT.java [hunk 1]
```diff
@@ -157,6 +159,31 @@ public void insertsWithoutPks() throws Exception {
...
+        Testing.Print.enable();
+
+        try (final JdbcConnection connection = databaseConnection()) {
+            populateTable(connection, "s1.anumeric");
+        }
+        startConnector();
+
+        sendAdHocSnapshotSignal("s1.anumeric");
+
+        Thread.sleep(5000);
```
Comment (gunnarmorling) [inline debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/IncrementalSnapshotIT.java:173]: Can we avoid this, in favour of something based on Awaitility?

**Review by gunnarmorling — APPROVED** [review]
LGTM overall, one comment inline. I don't fully get the solution though, can you expand a bit in the commit message perhaps?

**Comment by jpechane** [comment]
@gunnarmorling Comment added.

File: debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/IncrementalSnapshotIT.java (hunk 1 above)
Comment (jpechane) [reply debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/IncrementalSnapshotIT.java]: Superfluous, line removed.

**Comment by gunnarmorling** [comment]
Excellent, thanks for the great comment, it helped a lot!

**Pull request merged**